.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
```

Comment:
Just revert to the above, but remove the 5.3 line. No need to specify the matrix if we just want to remove 5.3. I also don't want to include the php nightly at this point.

Reply:
I thought as much, but then got too busy yesterday to action. Doing so now :)